data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/common/types/Channel.kt

```diff
@@ -30,7 +30,11 @@ data class Channel(
         @JsonProperty("topic") val topic: Topic,
         @JsonProperty("purpose") val purpose: Purpose,
         @JsonProperty("previous_names") val previousNames: List<String>,
-        @JsonProperty("num_members") val numMembers: Int) {
+        @JsonProperty("num_members") val numMembers: Int,
```

Comment:
this has to be an optional

Reply:
does it really? the field was already included

Reply:
```When using the method with the include_num_members parameter, we return a num_members field```

https://api.slack.com/methods/conversations.info